### ISSUE TITLE
error when installation

### ISSUE THREAD

**Issue by zhengguanji** [jonaprieto/flask-ponywhoosh#19]
There was an error when I use `pip install flask-ponywhoosh` to install the extension, the error as follows:
```
Could not find a version that satisfies the requirement flask-ponywhoosh (from versions: )
No matching distribution found for flask-ponywhoosh
```

**Comment by zhengguanji** [comment]
My Python version is 3.4.

**Comment by jonaprieto** [comment]
The problem is Flask-PonyWhoosh is only available for Python 2.7.X. But supporting Python 3.4 shouldn't be a problem because Flask now support that version, and we can work with it.

**Comment by zhengguanji** [comment]
I've tried again in Python virtual envrionment that use Python 2.7, and I got the same error. Why?

**Comment by jonaprieto** [comment]
It was a bug with pypy and other stuffs related with the name of the repository. I solved the problem in ed675b1762a69f0b787924dbade3be3a004ff5ce. Please try it again.

```bash
$ pip install flask-ponywhoosh
```

**Comment by zhengguanji** [comment]
It can be installed properly, thanks.
